docs/application/resource.md

```diff
@@ -57,33 +77,35 @@ Resources can depend on each other by using output references. The referenced en
 For example, if a container service depends on a MySQL resource, the container service can reference the MySQL endpoint in its environment variable as `${res.my-db-name.address}`.
 The output reference is in the following format: `${res.<Resource Name>.<Resource Output Name>}`.
 
-![resource-ref](/img/v0.5.0/application/resource/resource-ref.png)
+![resource-ref](/img/v0.6.0/application/resource/resource-ref.png)
 
 ## Rollback Resource
 
 1. Navigate to the `Resources` tab in the environment view.
 2. Find the resource you need to rollback.
 3. Click on the operation drop-down button, select `Rollback`.
 
-![resource-rollback](/img/v0.5.0/application/resource/resource-rollback.png)
+![resource-rollback](/img/v0.6.0/application/resource/resource-rollback.png)
 
 4. Choose the historic version you need to roll back to.
 5. Confirm the configuration changes for the rollback operation, click the `Rollback` button to complete the rollback operation.
 
-![resource-rollback2](/img/v0.5.0/application/resource/resource-rollback2.png)
+![resource-rollback2](/img/v0.6.0/application/resource/resource-rollback2.png)
+
+> You can also rollback with preview before applying the changes.
```

Comment:
You can also preview a rollback before applying the changes.

Reply:
updated.